proton/proton.cfg

```diff
@@ -1 +1,6 @@
-{"key":"user.pub","vixen_folder":"~/Dropbox/LightShow/Sequencing/Sequence Data"}
\ No newline at end of file
+{
+  "key": "tests2017.pub",
+  "vixen_folder": "/home/lightshow/Dropbox/Light Show 2017/Sequencing/Sequence Data",
```

Comment:
using `~` or `$HOME` is more reliable since this assumes that the user will be named 'lightshow', but as long as it always runs on the same computer (or one of our computers) this won't blow up.